conbench/logger.py

```diff
@@ -0,0 +1,94 @@
+"""
+Set up Python's logging infrastructure for the web application.
+
+Acquire root logger instance and populate it with handlers, applying a certain
+log format. This takes effect for Conbench code as well as integrated
+libraries, such as SQLAlchemy
+
+A package or module within Conbench should obtain and use a logger instance in
+the 'conbench' sub-tree of the logging namespace hierarchy, via e.g.
+
+    log = logging.getLogger('conbench.xxx')
```

Comment:
Is this equivalent to `log = logging.getLogger(__name__)` or are there exceptions? If it is equivalent, maybe we should recommend using `__name__` here so it's more future-proof if we rename/reorganize the files.

Reply:
There might be exceptions when `__name__` becomes too long :). I like the idea of recommending `__name__` but after all logs should be useful and what we consider useful in this context might change over time!